packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -374,3 +393,83 @@ export enum PersonaInitialsColor {
    */
   transparent = 15,
 }
+
+/**
+ * An object with each Key correlating to a value from PersonaPresence Enum.
+ */
+export type TPresenceDescriptors = {
+  [PersonaPresence.none]?: string;
+  [PersonaPresence.online]?: string;
+  [PersonaPresence.offline]?: string;
+  [PersonaPresence.away]?: string;
+  [PersonaPresence.dnd]?: string;
+  [PersonaPresence.busy]?: string;
+  [PersonaPresence.blocked]?: string;
+};
+
+export const defaultPresenceDescriptors: TPresenceDescriptors = {
+  [PersonaPresence.offline]: 'Offline',
```

Comment:
Our policy so far has been no hardcoded strings.  E.g. a customer localizes in 'fr', does this mean they have to download all the en strings too?

I'm open to having a better policy.

Reply:
That's fair, but without something set as default, this feature would be forced to opt-in and wouldn't help accessibility unless somebody uses the presenceDescriptors prop. How should I handle it?

Reply:
1. Expose the props.
2. Show correct usage in examples.
3. Once we have a localization pipeline in place which can allow us to bundle strings with the library, I'm good with adding much better defaults.

It's possible we could discuss separating out strings in a resjson file and consume that until we have a mechanism for replacing them with localized resources.

Reply:
Slighly forking the thread a bit, but what is the rough estimate for the loc pipeline @dzearing ?

Reply:
We had a meeting about this today actually! @aditima @iclanton It was mentioned in the meeting that "it wasn't a huge priority because other things were more important." I really am not sure that is true. It would be ideal to have localization working asap in Fabric so that we can avoid localizing differently across scenarios for things like datepickers and default formatting.

Reply:
I can't provide a timeline. But @iclanton was going to talk with @TheLarkInn about the right way for libraries to provide loc strings.